zoo-css-challenge/style.css

```
@@ -166,13 +216,23 @@ footer {
   background-color: rgba(255, 255, 255, 0.75);
   padding: 50px;
   max-width: 400px;
+  border-radius: 1.5rem;
 }
 
 .education-container h3,
 .education-container p {
   margin-bottom: 1em;
 }
 
+.education-container button{
+  font-size: 2rem;
+}
+
+
+.education-container button:hover,:focus{
```

```suggestion
.education-container button:hover, .education-container button:focus{
```
Like my other comment, we should update `:focus` to apply to only the buttons in the learning section